ios/Gekidou/Sources/Gekidou/Networking/Network+Users.swift

```diff
@@ -15,7 +15,7 @@ extension Network {
             // remove existing users in the database
             users = [User]()
             let storedUserIds = Database.default.queryUsers(byIds: userIds, forServerUrl: serverUrl)
-            if !(userIds.filter{ !storedUserIds.contains($0) }).isEmpty {
+            if !(userIds.subtracting(storedUserIds)).isEmpty {
```

Comment:
filter and contain on the Sets executes in O(n) while `subtracting` is O(1) thus making this much faster and removing the App Hang issue reported in Sentry